packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -61,54 +62,63 @@ export const MafsGraph = React.forwardRef<
     }));
 
     return (
-        <View
-            style={{
-                width,
-                height,
-                position: "relative",
+        <GraphStateContext.Provider
```

Comment:
I don't know why GH is being so silly about this file. I think the only thing I changes was wrapping existing things in the Provider.

Reply:
You can get a cleaner diff if you check the "hide whitespace" box:

<img width="692" alt="Screen Shot 2024-04-01 at 11 14 30 AM" src="https://github.com/Khan/perseus/assets/693920/12dd1d61-3475-437b-815e-7c968717b528">